tools/stonith_admin.c

```diff
@@ -528,6 +528,12 @@ main(int argc, char **argv)
         target = options.unregister_level;
     }
 
+    if (options.timeout > INT_MAX / 1000 || options.timeout < INT_MIN / 1000) {
+        printf("stonith_admin: Integer value \"%d\" for -t out of range\n", options.timeout);
```

Comment:
Simply replace `printf` with `out->err` (that formats the message as text or XML depending on the value of `--output-as`)

We don't need the `stonith_admin: ` prefix